cpp_exercise/src/lib/filehandlerlib.h

```diff
@@ -0,0 +1,46 @@
+#ifndef FILEHANDLERLIB_H
+#define FILEHANDLERLIB_H
+
+
+#include <string>
+#include <vector>
+#include <iostream>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <cstring>
+#include <sys/types.h>
+
+
+
+
+/**
+ * @brief FileHandler Class for creating, writing, reading, and removing
+ * 
+ */
+
+class FileHandler{
+private:
+const std::string _file_name; //input file name
+uint8_t _fd;  //file desriptor
```

Comment:
are you sure about uint8?  

When I read the documentation all I see about file descriptor is "a nonnegative integer".

Reply:
Fixed